docs/operation_center/integration_catalog/email/vade.md

```diff
@@ -0,0 +1,24 @@
+ # 1. Overview
+ 
+Vade for M365 offers AI-based protection against dynamic, email-borne cyberattacks targeting Microsoft 365.  
+API-based, Vade for M365 offers a native Microsoft 365 user experience and catches 10x more advanced threats than Microsoft.  
+
+In this documenation we will explain how to collect and send Vade for M365 logs to SEKOIA.IO.
+
+# 2. Configure
+
+First you need to reach the `Playbooks` page in order to initiate your playbook using the dedicated button.  
+
+You can directly choose the `Get M365 Email Events` trigger if you are creating a playbook from scratch otherwise you will have to find it
+within the `Actions library` panel under the `Vade secure` menu to drag and drop the trigger on the graph.  
+
+To start configuring the selected trigger, you'll need to bring Vade's documentation which can be found [here](https://m365.eu.vadesecure.com/docs/).
+This documentation will allow you to enter module information such as `api_host`, `client_id`, `client_secret` and `oauth2_authorization_url`.  
```

Comment:
```suggestion
This documentation will allow you to get the following information: your `client_id` and your `client_secret`. You can also get the `api_host` and `oauth2_authorization_url`, if necessary.
```
@LeoArurault, Can we have default value for `api_host` and `oauth2_authorization_url` in the manifest of the VadeSecure module?

Reply:
The manifest already suggest `https://m365.eu.vadesecure.com` for the `api_host`. But not for the `oauth2_authorization_url` and I'm not sure about the default value of this field